extensions/control-plane/transfer/transfer-pull-http-dynamic-receiver/README.md

```diff
@@ -0,0 +1,36 @@
+# Http Dynamic EDR receiver
+
+This extension is similar to the HTTP EDR receiver. The difference is that the URL is not configured
+at startup time, but instead the callback url is provided to the consumer when initiating a transfer request 
+by passing a custom property in the transfer request payload e.g. The URL will be stored in the transfer process
+and will be used by the consumer to dispatch the EDR
+
+```json
+{
+  "edctype": "dataspaceconnector:datarequest",
+  "protocol": "ids-multipart",
+  "assetId": "test-document",
+  "contractId": "1:8147d6d6-9734-4821-b93d-2832ea7892e4",
+  "dataDestination": {
+    "type": "HttpProxy"
+		
+  },
+  "transferType": {
+    "contentType": "application/octet-stream",
+    "isFinite": true
+  },
+  "managedResources": false,
+  "connectorAddress": "http://localhost:8282/api/v1/ids/data",
+  "connectorId": "consumer",
+  "properties": {
+    "receiver.http.endpoint" : "http://localhost:9999"
```

Comment:
That was my first implementation but since the `dataDestination` it's sent and stored in the provider side also i was not sure if it was a leak on private information (consumer side)

Reply:
Oh, you're right, what a pity, this is resolved :ok_hand: